src/v2/Artsy/SystemContext.tsx

```diff
@@ -103,6 +104,7 @@ export const SystemContextProvider: SFC<SystemContextProps> = ({
     relayEnvironment,
     user,
     setUser,
+    isEigen: sd.EIGEN,
```

Comment:
isEigen is already a prop in the same file but wasn't being set
https://github.com/artsy/force/blob/741dec64e9828067b3f1fce45dfa664862a868c9/src/v2/Artsy/SystemContext.tsx#L49